.gitignore

```diff
@@ -1,7 +1,6 @@
 launch.json
 settings.json
 models/
-data/
```

Comment:
The reason we added data folder to gitignore is sometimes the example will download test data to `data` folder: https://github.com/microsoft/Olive/blob/main/examples/cifar10_openvino_intel_hw/config.json#L27. If you want to reuse `data`, can you please double check our examples and rename their data folder?

Reply:
Thanks Xiaoyu. There is an item `examples/**/data` which can ignore the data under examples. So it is fine.